packages/ETLoad.package/ETLOopMapTest.class/instance/testConversionToSmallInteger.st

```diff
@@ -2,4 +2,6 @@ testing
 testConversionToSmallInteger
```

Comment:
I would prefer a few more tests, for "interesting numbers": 0, -2, 1, -0x3FFFFFFF, 16r7FFFFFFE.